Field cleanup

davidrapson opened the pull request:
Re-do of #142. Both removes references to old trail images and begins the process of cleaning up the hero image fields but also uses the new project config to remove old hero image fields, and any other unused fields from the admin.

mattandrews commented on lib/ContentHelpers.php:
```diff
@@ -25,8 +25,8 @@ public static function getCommonFields(Entry $entry, $status, $locale)
-            'hero' => $entry->heroImage ? Images::extractHeroImage($entry->heroImage) : null,
-            'heroCredit' => $entry->heroImageCredit ?? null,
+            // @TODO: Remove heroNew once main app is updated
+            'hero' => Images::buildHero($entry->hero),
```
This might fix this issue, too: https://github.com/biglotteryfund/blf-alpha/issues/1748

davidrapson replied:
This change won't, but this should: https://github.com/biglotteryfund/blf-alpha/pull/1722/files#diff-981f3f59b5bbe0747c50a0bf4457da5eR62

davidrapson commented:
@mattandrews Without this becoming a mega PR…I've updated this to removed the unused `caseStudies` section and fields (we replaced it with "project stories") and to remove the unused `homepageHeroImages` field.
